src/js/apps/forms/form/form_app.js

```diff
@@ -81,6 +82,7 @@ export default App.extend({
     this.responses = action.getFormResponses();
     this.latestResponse = latestResponse;
     this.isReadOnly = action.isLocked() || form.isReadOnly();
```

Comment:
My assumption is that `action.isLocked()` (based on `!!action.locked_at`) is separate from this permission. In that the reason for locking an action is not related to what permissions the user has.

So if that's not the case, I'll need to rework some things.

Reply:
I've updated this in the last push, let me know if I missed anything 🤔